app/src/main/java/ru/netology/fmhandroid/dto/Note.kt

```diff
@@ -1,15 +1,25 @@
 package ru.netology.fmhandroid.dto
 
+import java.time.LocalDateTime
+
 data class Note(
     val id: Int,
-    val patientId: Int,
+    val patientId: Int? = null,
     var description: String,
-    val creatorId: Int,
-    val executorId: Int,
-    val createDate: String,
-    val planeExecuteDate: String,
-    val factExecuteDate: String,
-    val statusId: Int,
-    var comment: String,
-    val deleted: Boolean
-)
\ No newline at end of file
+    val creatorId: Int? = null,
+    val executorId: Int? = null,
+    val createDate: String? = null,
+    val planeExecuteDate: LocalDateTime? = null,
+    val factExecuteDate: LocalDateTime? = null,
+    val noteStatus: NoteStatus? = null,
+    var comment: String? = null,
+    val deleted: Boolean = false,
+    val shortExecutorName: String,
+    val shortPatientName: String
+)
+
+enum class NoteStatus {
+    ACTIVE,
+    CANCELED,
+    EXECUTED
+}
```

Comment:
Я бы во внутрь `Note` затащил для большего выделения принадлежности одного класса к другому.
```suggestion
) {
    enum class Status {
        ACTIVE,
        CANCELED,
        EXECUTED
    }
}
```

Reply:
Исправлено